### DISCUSSION TITLE
What is the University GitHub Enterprise Address?

### DISCUSSION THREAD

**Discussion by karres-illinois** [uillinois-community/uillinois-community.github.io#50] [Q&A] [answered]
Hi

I am trying to use GitHub Desktop from a non-University PC.  It is asking for the "Enterprise Address" when trying to login.  What is that address?

**Comment by jtgorman** [comment, marked as answer]
There's not an "Enterprise Address". Some clients allow the option to connect to github.com or to a local github server. You'll want to make sure to connect to github.com.

**Comment by karres-illinois** [comment]
Thanks Jon


--
Dean Karres
Sr Research Programmer, Beckman Institute
Illinois ACCESS Campus Champion
https://campuschampions.web.illinois.edu/
